src/views/QueryEditorPullRequestReviews.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Input, Select, InlineField } from '@grafana/ui';
+import { SelectableValue } from '@grafana/data';
+import { RightColumnWidth, LeftColumnWidth } from './QueryEditor';
+import { PullRequestTimeField } from '../constants';
+import type { PullRequestReviewsOptions } from '../types/query';
+
+interface Props extends PullRequestReviewsOptions {
+  onChange: (value: PullRequestReviewsOptions) => void;
+}
+
+const timeFieldOptions: Array<SelectableValue<PullRequestTimeField>> = Object.keys(PullRequestTimeField)
+  .filter((_, i) => PullRequestTimeField[i] !== undefined)
+  .map((_, i) => {
+    return {
+      label: `${PullRequestTimeField[i]}`,
+      value: i as PullRequestTimeField,
+    };
+  });
+
+const defaultTimeField = timeFieldOptions[0].value;
+
+const QueryEditorPullRequestReviews = (props: Props) => {
```

Comment:
This is a copy and paste of the Pull Request one as they're going to be identical form wise